### ISSUE TITLE
nncase ver 0.1 and ver 0.2 for windows are not working

### ISSUE THREAD

**Issue by easy-and-simple** [kendryte/nncase#142]
I want to covert my model to kmodel3 but nncase 0.1 is not working and gives error that padding doesn't match and gives error
Fatal: Layer Pad is not supported: Only paddings of [[0,0],[1,1],[1,1],[0,0]] is supported
It always return this error no matter what padding I set.
Maybe issue is with conversion of my model from keras to tflite because there you don't supply converter and I am using tensorflow converter that probably is no supported by your nncase as ver 0.1 as well ver 0.2 because bot cant convert my tflite file. nncase 0.2 throw error Fatal: KPU allocator cannot allocate more memory.
What converter I need to use to convert h5 keras file to tflite file that support your nncase 0.1 or 0.2?

**Comment by sunnycase** [comment]
0.1 is not maintained now.
For `Fatal: KPU allocator cannot allocate more memory.` please refer to FAQ.

**Comment by easy-and-simple** [comment]
Maybe you should be more king, and to make some effort to find problem instead to send me to read FAQ
1. Version 0.1 is not supported, but I need to use it, because v0.2 that is supported can't convert to kmodel3 that runs more that 3 times faster on K210 than kmodel4
2. There are cases when nncase v0.1 convert model without issues, and then model loads and runs on K210, while for same model SUPPORTED nncase v0.2 throws error Fatal: KPU allocator cannot allocate more memory. Obviously SUPPRTED v0.2 is not working and issue is not ralated to 2MB RAM doe KPU?!

**Comment by sunnycase** [comment]
1. How do you get this conclusion?
2. 0.1 has a bug processing paddings that does incorrect padding fusion. It will produce fast but incorrect models. 0.2 **correctly** processed paddings which may make original conv2ds a little larger. that may not fit in KPU memory 

You didn't upload your model so debugging is down to you
It will save much time to do a simple search https://github.com/kendryte/nncase/issues?q=Allocator and do these checks before opening this issue.

**Comment by easy-and-simple** [comment]
Bug or not but model have successfully loaded to maixpy board. I deleted this model because it was not trained, but it worked and draw bounding boxes with x=y=0 for 1 class no matter what is in image, so problem is with nncase 0.2
Also I still can't understand what is the problem with uint8? Is it very hard to you to do uint8/255.0?
